[KB-273] Rewrite problem formulation to include the challenges (#206)
chhoumann/thesis-chemcam: report_thesis/src/sections/problem_definition.tex

Due to the high cost of data collection, datasets are often small. This limits the number of samples available for evaluation, affecting the generalizability and robustness of the models\cite{p9_paper}.

\subsection{Problem Formulation}

> The objective of this research is to develop a computational model, denoted as $\mathcal{F}: \mathbb{R}^N \rightarrow \mathbb{R}^{n_o}$, to predict major oxide concentrations in geological samples from processed \gls{libs} spectral data.
> This task is complicated by the high dimensionality of the data, multicollinearity among spectral features, matrix effects that alter emission intensities, and the limited availability of data.

Author: ```suggestion
The objective of this research is to develop a computational model, denoted as $\mathcal{F}: \mathbb{R}^N \rightarrow \mathbb{R}^{n_o}$, to predict major oxide concentrations in geological samples from processed \gls{libs} spectral data, while overcoming the challenges posed by the high dimensionality of the data, multicollinearity among spectral features, matrix effects altering emission intensities, and the limited availability of data.
```
